### DISCUSSION TITLE
中大語言課修課經驗(2)

### DISCUSSION THREAD

**Discussion by THH1118** [1101-GS4719-C/Python-Community-GS4719-C#1755] [General]
日文一(B) 洪韶翎 老師

老師開的課比較少，印象中只有1A跟1B，老師的日文很好，教學的內容不會按照課本的來(但還是會稍微帶一點課本的內容，但不多)，老師會傾向教一些實際跟日本人比較用得到的對話，也常常會介紹一些日本人講話的習慣，比較適合可以自學文法想要學一些更進階的同學來修。

考試的話平常會有口語練習跟口說小考，也有期中跟期末考，老師的考試不會出課本的句子，會用到一些課本沒有的單字，但老師很鼓勵大家創意，所以題目都沒有標準答案，如果你寫出來的不是正解但解釋的通也是能拿到分數的，考試也有加分題，我自己拿的分數也是有90up，但要修這堂課可能要先評估一下自己適不適合老師的教學風格，跟其他的日文老師差異會蠻大的，但也會覺得很新鮮。

**Comment by WeiYen1103** [comment]
我有上這堂課，我覺得老師人很好，但洪老師考試相較其他老師是真的有一點不同，但這堂課我有點抓不到老師的評分方式，所以最後沒有高分過QQ
